src/tests/ftest/posix/simul.py

```diff
@@ -0,0 +1,153 @@
+        Test #24: lseek, individual mode.
+        Test #25: read, individual mode.
+        Test #26: write, individual mode.
+        Test #27: chdir, individual mode.
+        Test #28: directory stat, individual mode.
+        Test #29: statfs, individual mode.
+        Test #30: readdir, individual mode.
+        Test #31: mkdir, individual mode.
+        Test #32: rmdir, individual mode.
+        Test #33: unlink, individual mode.
+        Test #34: rename, individual mode.
+        Test #35: creat, individual mode.
+        Test #36: truncate, individual mode.
+        Test #37: symlink, individual mode.
+        Test #38: readlink, individual mode.
+        Test #39: link to one file, individual mode.
+        Test #40: link to a file per process, individual mode.
+        Test #41: fcntl locking, individual mode.
+
+    :avocado: recursive
+    """
+
+    def run_simul(self, include=None, exclude=None):
+        """ Run simul
+
+        include str:  comma-separated list of tests to include
+        exclude str:  comma-separated list of tests to exclude
+
+        If include value is set, exclude value is ignored and vice versa.
+
+        """
+        simul_openmpi = "/usr/lib64/openmpi3/bin/simul"
+        simul_mpich = "/usr/lib64/mpich/bin/simul"
+        simul_dict = {"openmpi": simul_openmpi,
+                      "mpich": simul_mpich
+                      }
+
+        clients = self.params.get("test_clients", '/run/hosts/*')
+        dfuse_mount_dir = self.params.get("mount_dir", '/run/dfuse/*')
+
+        if not self.pool:
+            self.log.info("Create a pool")
+            self.add_pool()
+        if not self.container:
+            self.log.info("Create container")
+            self.add_container(self.pool)
+
+        self.start_dfuse(self.hostlist_clients, self.pool, self.container)
+        self.dfuse.check_running()
+
+        # Load openmpi
+        for mpi, simul in simul_dict.items():
+            load_mpi(mpi)
+
+            for client in clients:
+                if include:
+                    cmd = "/usr/bin/ssh {0} {1} -vv -d {2} -i {3}".format(
+                        client,
+                        simul,
+                        dfuse_mount_dir,
+                        include)
+                else:
+                    cmd = "/usr/bin/ssh {0} {1} -vv -d {2} -e {3}".format(
+                        client,
+                        simul,
+                        dfuse_mount_dir,
+                        exclude)
+                try:
+                    run_command(cmd)
+                except DaosTestError:
+                    self.stop_dfuse()
+                    self.fail("Simul failed on {}".format(mpi))
+
+                self.log.info("Simul passed on %s", mpi)
+
+        self.stop_dfuse()
+
+    def test_posix_simul(self):
+        """Test simul.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=posix,simul
```

Comment:
Going from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-5958/5/testReport/FTEST_posix/ this test took less than two minutes, therefore I think it would be worth enabling by default.  You should have a dfuse tag here as well.

Reply:
Will do

Reply:
@ashleypittman , do you know if there is a env variable that I can use to set caching on/off?

Reply:
It'll be on by default, or there's a dfuse setting (in the dfuse_utils.py module) to disable it.